dispatch/generic_static/module.modulemap

```diff
@@ -3,7 +3,6 @@ module Dispatch {
 	export *
 	link "dispatch"
 	link "BlocksRuntime"
-	link "DispatchStubs"
```

Comment:
Not a request for changes, but by removing this line, this file is identical to https://github.com/swiftlang/swift-corelibs-libdispatch/blob/main/dispatch/generic/module.modulemap, so we might be able to remove the static case from the vfs overlay generation https://github.com/swiftlang/swift-corelibs-libdispatch/blob/84ec97ce04d050bdc95c2b6480397e43911afe4b/dispatch/CMakeLists.txt#L6-L7.